### ISSUE TITLE
speed 

### ISSUE THREAD

**Issue by kaishijeng** [daijifeng001/MNC#4]
It is quite slow to process one frame, like 300-400ms/fame on a titanX card.
R-FCN is faster from the paper, but I am not able to make it working because of lacking Matlab.
Just wondering how much speed will improve if I use Alexnet or Squeezenet instead of VGG16

Thanks,

**Comment by Oh233** [comment]
We didn't test the speed of Alexnet. But you may reach ~160ms/frame on a TitanX card if you use ZF net.

**Comment by kaishijeng** [comment]
Do you have network arch, test.prototxt and pre-trained model of ZF net
available so that I can try it?

Thanks

**Comment by qinhaifangpku** [comment]
@Oh233 hi~  i am train on my own data. about 12s/iter on K80, is it something wrong happend?

**Comment by Oh233** [comment]
@qinhaifangpku 

I use iter_size = 8 in the solver, in order to simulate the 8gpu training described in paper. Thus the time should be 1.5s/iter in your case. Maybe that is a reasonable time for K80 because I didn't have such GPU to test.

If you are interested, you may set a timer to see which part cause the slow speed. (e.g. preparing data in CPU or the network forward)

**Comment by qinhaifangpku** [comment]
@Oh233 thank you !  i set a timer in mnc_data_layer to see which part to slow speed but in show that get_next_minibatch did not take too long.  i wonder if i can see excute time in every layer?  my data only has too much mask which different from yours.
